spyderlib/spyder_restart.py

```diff
@@ -0,0 +1,123 @@
+# -*- coding: utf-8 -*-

```

Comment:
Wouldn't it better to call this file `restart_spyder.py`, or even better simply `restart.py`?